scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -428,8 +428,13 @@ def dijkstra(csgraph, directed=True, indices=None,
         will decrease computation time by aborting calculations between pairs
         that are separated by a distance > limit. For such pairs, the distance
         will be equal to np.inf (i.e., not connected).
-
-        .. versionadded:: 0.14.0
+    multi_target: bool, optional
+        if False (default), independantly find the shortest path
+        to every index in indices
+        If True, find the shortest path to any index in indices,
+        resulting distance matrix with be Kx1 vector, where K is the len(indices)
+        and predecessor matrix will be Nx1. Much faster if this is what you want.
+        .. versionadded:: 0.15.0
```

Comment:
A few typos in this doc block, and also the `versionadded` was for another argument so shouldn't have been modified (and we aren't on version `0.15.0` either). If the feature does get added you'd probably want a new `versionadded` for i.e., `1.3.0`.

Reply:
I think i've addressed the typos in the latest changes, and altered the version to 1.3.0. Please let me know wha the accepted protocol for that is.